packages/augur-ui/src/modules/trading/components/form/form.styles.less

```diff
@@ -363,8 +365,8 @@
       border-radius: 0;
       bottom: 0;
       left: 0;
-      padding-bottom: 1rem;
-      padding-top: 1rem;
+      padding-bottom: @size-16;
+      padding-top: @size-16;
```

Comment:
merge these into a single `padding: @size-16 0;` so it's only one line/less chars